docs/en/pact-functions.rst

```diff
@@ -1938,6 +1900,48 @@ are base-16 strings of length 32.
 Guards
 ------
 
+create-capability-guard
+~~~~~~~~~~~~~~~~~~~~~~~
+
+*capability* ``-> bool`` *→* ``guard``
+
+Creates a guard that will enforce that CAPABILITY is acquired.
+
+.. code:: lisp
+
+   (create-capability-guard (BANK_DEBIT 10.0))
+
+create-capability-pact-guard
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*capability* ``-> bool`` *→* ``guard``
+
+Creates a guard that will enforce that CAPABILITY is acquired and that
+the currently-executing defpact is operational.
+
+.. code:: lisp
+
+   (create-capability-pact-guard (ESCROW owner))
+
+create-module-guard
```

Comment:
Perhaps we want to drop these from the documentation altogether now?

Reply:
We have a warning for now that corresponds with these natives. I think they should be documented as long as they're supported. It's easy enough to delete them in the next release and regenerate the documentation with the removed entries.